python_modules/dagster/dagster/_core/definitions/declarative_automation/automation_context.py

```diff
@@ -214,17 +218,19 @@ def legacy_context(self) -> LegacyRuleEvaluationContext:
             "Cannot access legacy context unless evaluating a condition containing a RuleCondition.",
         )
 
-    @property
-    def previous_requested_subset(self) -> Optional[EntitySubset[AssetKey]]:
-        """Returns the requested subset for the previous evaluation. If this asset has never been
+    def get_previous_requested_subset(
+        self, key: T_EntityKey
+    ) -> Optional[EntitySubset[T_EntityKey]]:
+        """Returns the requested subset for the previous evaluation. If the entity has never been
         evaluated, returns None.
         """
-        return (
-            self.asset_graph_view.get_subset_from_serializable_subset(
-                self._cursor.previous_requested_subset
-            )
-            if self._cursor
-            else None
+        if self._daemon_cursor is None:
```

Comment:
Why would this be none? First tick?

Reply:
exactly yep

Reply:
oops no you're right this is never None, updated upstack